sound/soc/intel/boards/sof_sdw_rt1308.c

```diff
@@ -132,6 +132,10 @@ int sof_sdw_rt1308_init(const struct snd_soc_acpi_link_adr *link,
 			struct sof_sdw_codec_info *info,
 			bool playback)
 {
+	/* Count amp number and do init on playback link only. */
+	if (!playback)
+		return 0;
```

Comment:
[rt1308_sdw_dai[]](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/codecs/rt1308-sdw.c#L626) doesn't have .capture supported. Do we still need the test? @shumingfan Do you know if rt1308 supports feedback by HW?

Reply:
> [rt1308_sdw_dai[]](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/codecs/rt1308-sdw.c#L626) doesn't have .capture supported. Do we still need the test? @shumingfan Do you know if rt1308 supports feedback by HW?

The RT1308 could support the feedback. But, the driver doesn't support yet.

Reply:
Right, that was the idea. Both RT1308 and RT1316 can support feedback, so at the machine driver level we want to make sure the init is only done once. However if we want to enable feedback there are additional steps needed (codec driver + dailink needed). that can be done later, if and when it's needed. For now I just wanted to have 1:1 mapping between the two machine driver inits.